README.md

```diff
@@ -132,10 +147,10 @@ You configure ExternalDNS later in the process.
     export CK8S_K8S_INSTALLER=[kubespray|capi] # set this to whichever installer was used for the kubernetes layer
     ```
 
-    > [!NOTE]
-    > The `air-gapped` flavor has a lot of the same settings as the `prod` flavor but with some additional variables that you need to configure yourself (these are set to `set-me`).
+> [!NOTE]
+> The `air-gapped` flavor has a lot of the same settings as the `prod` flavor but with some additional variables that you need to configure yourself (these are set to `set-me`).
 
-1. Then set the path to where the ck8s configuration should be stored and the PGP fingerprint of the key(s) to use for encryption:
+2. Then set the path to where the ck8s configuration should be stored and the PGP fingerprint of the key(s) to use for encryption:
```

Comment:
Can you please clarify why we are switching from automatic numbering to manual numbering and then back again to automatic?
It is a bit confusing and I would like to know the reason for my learning purpose.

Reply:
In order for the notes to actually render correctly they can't be indented and when I moved them back the automatic numbering broke.